### ISSUE TITLE
Allow overriding device class: Shutter & more missing

### ISSUE THREAD

**Issue by KNXBroker** [home-assistant/frontend#10873]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

There is a new functionality which allows overriding device classes (https://github.com/home-assistant/frontend/pull/10777).
I'm missing the selection shutter. Furthermore there should be a 'X' to undo a selection.

### Describe the behavior you expected

Shutter selection and a 'X' for removal of the selection should be available. 

### Steps to reproduce the issue

1. Setup cover group
2. Select cover group
3. Go to settings

### What version of Home Assistant Core has the issue?

core-2021.12.0b7

### In which browser are you experiencing the issue with?

Chrome Version 96.0.4664.93

### Which operating system are you using to run this browser?

Windows 10

### State of relevant entities

```txt
cover.zentral_rollladen

Zentral Rollladen
open	

current_position: 8
entity_id: cover.zentral_rollladen_og, cover.zentral_rollladen_eg
assumed_state: true
friendly_name: Zentral Rollladen
supported_features: 79
```

**Comment by KNXBroker** [comment]
By the way, having this functionality for switch device classes would be appreciated as well

**Comment by mvdwetering** [comment]
It is not only Shutter that is missing (at least for me on a covergroup). I only see Window, Door, Garage Door and Gate. I would expect all from https://www.home-assistant.io/integrations/cover/#device-class

![image](https://user-images.githubusercontent.com/732514/152569884-224ea242-68d4-4c8c-80c8-ef302f2f9f57.png)

**Comment by KNXBroker** [comment]
Fixed by #11874 and #12042 . Thank you
